README.md

```diff
@@ -1,14 +1,163 @@
-# Getting started
+# How To Run Kasia Locally
+
+## Getting started
+
+Make sure you have the latest version of git downloaded - https://git-scm.com/downloads
+
+Verify your installation by running in your terminal:
+```
+git version
+```
+
+If you see a version number you have installed git successfully.
+
+## Install Rust
+  <details>
+  <summary>On Linux</summary>
+
+  1. Install general prerequisites
```

Comment:
I'm not sure these system dependencies are required in order to build the `cipher` wasm package.

I believe you took these dependencies from `Kaspa` node, is it correct?

I'll try to build the wasm package from a fresh environment to spot the required dependencies if any